libraries/AP_RangeFinder/AP_RangeFinder_MaxsonarSerialLV.cpp

```diff
@@ -0,0 +1,111 @@
+// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+/*
+ * Copyright (C) 2016  Intel Corporation. All rights reserved.
+ *
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_RangeFinder_MaxsonarSerialLV.h"
+#include <AP_SerialManager/AP_SerialManager.h>
+#include <ctype.h>
+
+#define MAXSONAR_SERIAL_LV_BAUD_RATE 9600
```

Comment:
If it's not needed in the header it's always better to put in the .cpp and avoid polluting the namespace. Rule is here is to have the smaller scope as possible.